README.md

```diff
@@ -0,0 +1,86 @@
+# dbt-rpc-client
+[![PyPI version](https://badge.fury.io/py/dbt-rpc-client.svg)](https://badge.fury.io/py/dbt-rpc-client)
+![PyPI - Status](https://img.shields.io/pypi/status/dbt-rpc-client)
+![PyPI - Python Version](https://img.shields.io/pypi/pyversions/dbt-rpc-client)
+![PyPI - License](https://img.shields.io/pypi/l/dbt-rpc-client)
+[![Build Status](https://travis-ci.com/goodeggs/dbt-rpc-client.svg?branch=master)](https://travis-ci.com/goodeggs/dbt-rpc-client.svg?branch=master)
+
+A python SDK for interacting with a [dbt rpc server](https://docs.getdbt.com/docs/rpc).
+
+## Installation
+
+```bash
+$ pip install dbt-rpc-client
+```
+
+## Basic Usage
+
+```python
+from dbt_rpc_client import DbtRpcClient
+
+rpc = DbtRpcClient(hostname="0.0.0.0", port=8580)
+
+# Getting the current status of the dbt rpc server.
+response = rpc.status()
+assert response.ok
+
+# Polling a dbt rpc operation.
+response = rpc.poll(request_token="1234abcd")
+assert response.ok
```

Comment:
it isn't clear if `response.ok` means the operation is done or the RPC call completed.

Reply:
@bobzoller `response` is a `requests.Response` object. The `ok` attribute on a `Response` object simply represents the HTTP status of the response.